molgenis-data/src/main/java/org/molgenis/data/semantic/Relation.java

```diff
@@ -1,34 +1,36 @@
 package org.molgenis.data.semantic;
 
-public enum Relation
-{
-	instanceOf("http://molgenis.org/biobankconnect/instanceOf"), link("http://molgenis.org/biobankconnect/link"), homepage(
-			"http://xmlns.com/foaf/0.1/homepage"), isDefinedBy("http://www.w3.org/2000/01/rdf-schema#isDefinedBy"), seeAlso(
-			"http://www.w3.org/2000/01/rdf-schema#seeAlso");
+public enum Relation {
 
+	instanceOf("http://molgenis.org/biobankconnect/instanceOf"), link(
+			"http://molgenis.org/biobankconnect/link"), homepage(
+			"http://xmlns.com/foaf/0.1/homepage"), isDefinedBy(
+			"http://www.w3.org/2000/01/rdf-schema#isDefinedBy"), seeAlso(
+			"http://www.w3.org/2000/01/rdf-schema#seeAlso"), hasLowerValue(
+			"http://molgenis.org/uml/hasLowerValue"), hasUpperValue(
+			"http://molgenis.org/uml/hasUpperValue"), isRealizationOf(
+			"http://molgenis.org/uml/isRealizationOf"), isGeneralizationOf(
+			"http://molgenis.org/uml/isGeneralizationOf"), hasSourceId(
+			"http://molgenis.org/uml/hasSourceId"), hasSourceName(
+			"http://molgenis.org/uml/hasSourceName");
+	
 	private String iri;
 
-	private Relation(String iri)
-	{
+	private Relation(String iri) {
```

Comment:
Can you fix the formatting? See https://github.com/molgenis/molgenis/blob/master/eclipse-java-molgenis-style.xml for the standard settings.